po/fr.po

```diff
@@ -1893,50 +1897,51 @@ msgstr "nom de l'attribut"
 
 #: rust/libnewsboat/src/filterparser.rs:64
 msgid "one of: =~, ==, =, !~, !=, <=, >=, <, >, between, #, !#"
-msgstr ""
+msgstr "L'un de: =~, ==, =, !~, !=, <=, >=, <, >, between, #, !#"
 
 #: rust/libnewsboat/src/filterparser.rs:65
 msgid "one of: quoted string, range, number"
-msgstr ""
+msgstr "L'un de: quoted string, range, number"
```

Comment:
Unlike "between" in the previous message, the "quoted string", "range" and "number" here are just words, and can be translated. I already filed an issue to clarify this better: #1354.

Also, there probably should be a space before the colon.